apps/bc-wallet-api-server/src/types/schema/index.ts

```diff
@@ -195,12 +195,12 @@ export type NewStepActionTypes =
   | NewChooseWalletAction
 
 export type Step = Omit<typeof steps.$inferSelect, 'asset'> & {
-  actions: StepActionTypes[]
+  actions?: AriesOOBAction[]
```

Comment:
Why was this reverted? We have 7 step types now

Reply:
Rolled back